### PR TITLE
Bau standardise deploy

### PR DESCRIPTION
**Pull request by robk-dluhc**
_Add ticket reference to Pull Request title: e.g. 'FS-123: Add content', if there is no ticket prefix with BAU_

### Change description
_A brief description of the pull request_

- [ ] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [ ] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
_If manual testing is needed, give suggested testing steps_

### CODE REVIEW

File: .github/workflows/copilot_deploy.yml [hunk 1]
```diff
@@ -208,62 +104,21 @@ jobs:
     uses: communitiesuk/funding-service-design-workflows/.github/workflows/post-deploy.yml@main
     with:
       run_performance_tests: ${{ inputs.run_performance_tests || true }}
-      run_e2e_tests: ${{ inputs.run_e2e_tests || false }}
-      environment: dev
+      run_e2e_tests_assessment: ${{ inputs.run_e2e_tests || false }}
+      run_e2e_tests_application: ${{ inputs.run_e2e_tests || false }}
```
Comment (srh-sloan) [inline .github/workflows/copilot_deploy.yml:108]: think it needs to have the new input names with appplication/assessment on the end

File: .github/workflows/copilot_deploy.yml [hunk 2]
```diff
@@ -276,62 +131,21 @@ jobs:
     uses: communitiesuk/funding-service-design-workflows/.github/workflows/post-deploy.yml@main
     with:
       run_performance_tests: ${{ inputs.run_performance_tests || false }}
-      run_e2e_tests: ${{ inputs.run_e2e_tests || true }}
+      run_e2e_tests_assessment: ${{ inputs.run_e2e_tests || true }}
+      run_e2e_tests_application: ${{ inputs.run_e2e_tests || true }}
```
Comment (srh-sloan) [inline .github/workflows/copilot_deploy.yml:135]: input names, as above

File: .github/workflows/copilot_deploy.yml [hunk 3]
```diff
@@ -344,59 +158,18 @@ jobs:
...
```
Comment (srh-sloan) [inline .github/workflows/copilot_deploy.yml:162]: input names

**Review by srh-sloan — CHANGES_REQUESTED** [review]

**Review by srh-sloan — APPROVED** [review]

File: .github/workflows/copilot_deploy.yml (hunk 2 above)
Comment (robk-dluhc) [reply .github/workflows/copilot_deploy.yml]: Amended

**Pull request merged**